library/nats

```diff
@@ -2,35 +2,38 @@ Maintainers: Derek Collison <derek@synadia.com> (@derekcollison),
              Ivan Kozlovic <ivan@synadia.com> (@kozlovic),
              Waldemar Salinas <wally@synadia.com> (@wallyqs)
 GitRepo: https://github.com/nats-io/nats-docker.git
-GitCommit: 710f0ed18645d78e97fa7fd8cdf9b80dbe936eb6
+GitCommit: 92ef9b356819df21c58695c26b29f57d8e5eb3a2
 
 Tags: 2.1.0-linux, linux
 SharedTags: 2.1.0, latest
 Architectures: amd64, arm32v6, arm32v7, arm64v8
-amd64-GitCommit: 710f0ed18645d78e97fa7fd8cdf9b80dbe936eb6
-amd64-Directory: amd64
-arm32v6-GitCommit: 710f0ed18645d78e97fa7fd8cdf9b80dbe936eb6
-arm32v6-Directory: arm32v6
-arm32v7-GitCommit: 710f0ed18645d78e97fa7fd8cdf9b80dbe936eb6
-arm32v7-Directory: arm32v7
-arm64v8-GitCommit: 710f0ed18645d78e97fa7fd8cdf9b80dbe936eb6
-arm64v8-Directory: arm64v8
+amd64-Directory: 2.1.0/scratch/amd64
+arm32v6-Directory: 2.1.0/scratch/arm32v6
+arm32v7-Directory: 2.1.0/scratch/arm32v7
+arm64v8-Directory: 2.1.0/scratch/arm64v8
 
-Tags: 2.1.0-nanoserver, nanoserver, nanoserver-1803
+Tags: 2.1.0-servercore1809, servercore1809
```

Comment:
To keep consistency with the rest of the official images, these should be `windowsservercore`

Reply:
Updated. :+1:

Reply:
Wait... :(
Oh, I see. We're gonna have to update our repo again.
Okay. One sec.

Reply:
@yosifkit can I get a review of changes on our side before we merge?
https://github.com/nats-io/nats-docker/pull/23

Just want to make sure it matches up with requirements on your end.

Reply:
Aaaaalllll righty. Rename done. CI green. :heavy_check_mark:

Reply:
Friendly ping, @yosifkit. Sorry we keep bothering you. Promise the next PR will go a lot smoother. :smile: :pray: